dotCMS/src/main/java/com/dotcms/publisher/bundle/business/BundleFactoryImpl.java

```diff
@@ -55,7 +56,7 @@ public void saveBundleEnvironment(Bundle bundle, Environment e) throws DotDataEx
     @Override
     public List<Bundle> findUnsendBundles ( String userId, int limit, int offset ) throws DotDataException {
 
-        List<Bundle> bundles = new ArrayList<Bundle>();
+        List<Bundle> bundles = new ArrayList<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'bundles' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)